src/main/java/frc/team2412/robot/subsystems/VisionSubsystem.java

```diff
@@ -25,12 +33,30 @@
  * <p>At some point we should discuss how we want to handle the different alliances.
  */
 public class VisionSubsystem extends SubsystemBase {
-	private PhotonCamera photonCamera;
-	private PhotonPipelineResult latestResult;
-	/** Null if no known robot pose, otherwise the last calculated robot pose from vision data. */
-	private Pose3d robotPose = null;
+	private static final boolean IS_COMP = Robot.getInstance().isCompetition();
+	// Rough measurements, origin is center of robot, +X is forward, +Y is left, +Z is up
+	public static final Transform3d ROBOT_TO_CAM =
+			IS_COMP
+					? new Transform3d(
+							new Translation3d(
+									// 6.5 inches from back of robot, back is -half of length (30 in.)
+									Units.inchesToMeters(-30.0 / 2 + 6.5),
+									// 7 inches from left, left is +half of width (26 in.)
+									Units.inchesToMeters(26.0 / 2 - 7),
+									// 29 inches above the ground
+									Units.inchesToMeters(29)),
+							// Camera has a slight yaw, -6.7 degrees following right-hand rule (thumb points to
+							// +Z/up, fingers curl in positive rotation (CCW looking down))
+							new Rotation3d(0, 0, Math.toRadians(-6.7)))
+					: new Transform3d(
+							new Translation3d(0, 0, Units.inchesToMeters(28)),
+							// Camera's upside down
+							new Rotation3d(Math.toRadians(180), 0, 0));
 
-	private SwerveDrivePoseEstimator poseEstimator;
+	private PhotonCamera photonCamera;
```

Comment:
Consider making this and `photonPoseEstimator` final, as they're only assigned once